AdobeIms/view/adminhtml/web/css/source/_module.less

```diff
@@ -0,0 +1,25 @@
+// /**
+//  * Copyright © Magento, Inc. All rights reserved.
+//  * See COPYING.txt for license details.
+//  */
+
+& when (@media-common = true) {
+  .adobe-sign-in-button {
+    float: right;
+    background: transparent;
+    border: none;
+    margin-right: 3%;
+    position: relative;
+    z-index: 99;
+    margin-top: -50px;
+  }
+  #adobe-ims-user-name {
```

Comment:
It's preferable to use classes instead of ids for CSS selectors

Reply:
ok